agent/alembic/versions/971cd06e3d76_create_pipline_notifications_table.py

```diff
@@ -0,0 +1,27 @@
+"""create pipline notifications table
```

Comment:
what is the purpose of this table? why we can't join directly with no_data_notifications?

Reply:
If we will extend pipeline notifications, it will be aggregated in notification attribute